### DISCUSSION TITLE
Dvwa-Login page

### DISCUSSION THREAD

**Discussion by CyberKaur** [projectdiscovery/nuclei#5027] [Q&A]
I tried to find vulnerability in dvwa login page using default dvwa-default-login.yaml template . I am getting Better Luck next time !

**Comment by dogancanbakir** [comment]
Please share repro details. Thanks.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
